### PR TITLE
Make configs more generic

### PR DESCRIPTION
**Pull request by ausmith**
## What does this PR change?
* Makes the templated configs more generic
* Removes company identifying data that is long since dead internally

## Does this PR relate to any other PRs?
* No

## How will this PR impact users?
* It won't

## Does this PR address any GitHub or Zendesk issues?
* No

## How was this PR tested?
* N/A

## Does this PR require changes to documentation?
* No

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Opencost release? If not, why not?
* Not a release impacting change, only modifying the configuration templates

### CODE REVIEW

File: configs/gcp.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "spotCPU": "0.006655",
     "RAM": "0.004237",
     "spotRAM": "0.000892",
-    "projectID": "guestbook-227502",
+    "projectID": "my-awesome-project",
```
Comment (michaelmdresser) [inline configs/gcp.json]: Is there a valid string we can provide that has invalid GCP project ID characters to guarantee no overlap with a valid ID? The same question goes for all fields; having something that is too short to be valid also counts, I think!

**Review by michaelmdresser — COMMENTED** [review]
You'll need to set up signoff for the DCO check to pass (requirement from CNCF to have it as a blocker). You can use the provided instructions in the DCO workflow link on this PR. In the longer term, I recommend setting up some sort of automation in your Git-interaction tool of choice to auto-signoff all commits or just commits to your fork of `opencost`.

File: configs/gcp.json (hunk 1 above)
Comment (ausmith) [reply configs/gcp.json]: So GCP project IDs must be 6-30 characters in length [source](https://cloud.google.com/resource-manager/docs/creating-managing-projects). AWS S3 bucket names must be between 3 and 63 characters in length [source](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html). And unfortunately AWS Athena table names can be anything less than 255 characters [source](https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html). Values chosen for each accordingly.

**Comment by nikovacevic** [comment]
cc @AjayTripathy can you confirm this won't have any unintended consequences?

**Review by michaelmdresser — APPROVED** [review]
Definitely should have Ajay take a look, but this LGTM. Thanks for addressing feedback as well as possible!

**Comment by AjayTripathy** [comment]
its fine, thanks!

**Review by AjayTripathy — APPROVED** [review]

**Pull request merged**